cloudant-client/src/main/java/com/cloudant/client/api/Database.java

```diff
@@ -559,15 +560,15 @@ public void deleteIndex(String indexName, String designDocId) {
      *
      * @param indexName   name of the index
      * @param designDocId ID of the design doc (the _design prefix will be added if not present)
-     * @param type        type of the index, defaults to "json"
+     * @param type        type of the index, valid values or "text" or "json"
```

Comment:
Should we say something here about "text" and "json" being the only valid options at this time?

Reply:
I think the javadoc makes that clear and is just another way of stating what you said.

If there is ever a new type of index, that will be documented here and in the Bluemix documents